tests/compositional/test_fluid_mixtures.py

```diff
@@ -0,0 +1,394 @@
+"""Module testing the creation of fluid mixtures and their mixins."""
+from __future__ import annotations
+
+from numpy import ndarray
+import pytest
+
+import numpy as np
+import porepy as pp
+import porepy.compositional as ppc
+
+
+@pytest.fixture(scope='module')
+def dummyeos():
+    """Dummy Eos"""
+
+    class DummyEos(ppc.AbstractEoS):
+
+        def compute_phase_state(
+            self, phase_type: int, *thermodynamic_input: ndarray
+        ) -> ppc.PhaseState:
+            pass
+
+    return DummyEos
+
+
+class MockModel(ppc.FluidMixtureMixin, ppc.CompositionalVariables, pp.BoundaryConditionMixin):
```

Comment:
I don't think this is used, expcept as a type hint?

Reply:
Yes, that is the reason. It really helps in the annotation of tests, especially during refactoring.
I was missing that in multiple other tests for the equation system for example.
Is it okay if we leave it in the first period while the code might undergo significant changes in naming and signatures?

Reply:
Good point. Let's keep it